indico/MaKaC/webinterface/pages/main.py

```diff
@@ -25,22 +25,17 @@
 
 
 class WPMainBase(base.WPDecorated):
+    sidemenu_option = None
 
-    def _createMenu( self ):
-        self._showAdmin = self._isAdmin or self._isCategoryManager
-
-    def _setCurrentMenuItem( self ):
-        return
-
-    def _display( self, params ):
-        sideMenu = self._getSideMenu()
-        self._setCurrentMenuItem()
+    def _display(self, params):
+        target = self._rh.getTarget()
 
         # Check if user is administrator
         self._isAdmin = session.user and session.user.is_admin
-        self._isCategoryManager = self._isAdmin or \
-                                  isinstance(self._rh.getTarget(), Category) and self._rh.getTarget().canModify(self._getAW()) or \
-                                  isinstance(self._rh.getTarget(), Conference) and self._rh.getTarget().getOwnerList()!=[] and self._rh.getTarget().getOwner().canModify(self._getAW())
+        self._isCategoryManager = (
+            self._isAdmin or
+            isinstance(target, Category) and target.canModify(self._getAW()) or
```

Comment:
I know you just reformatted it, but I'd add parentheses around the OR'd items which contain ANDs